src/sql/src/statements/statement.rs

```diff
@@ -89,6 +91,37 @@ pub enum Statement {
     ShowVariables(ShowVariables),
 }
 
+impl Display for Statement {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Statement::Query(s) => s.inner.fmt(f),
+            Statement::Insert(s) => s.inner.fmt(f),
+            Statement::Delete(s) => s.inner.fmt(f),
+            Statement::CreateTable(s) => s.fmt(f),
+            Statement::CreateExternalTable(s) => s.fmt(f),
+            Statement::CreateTableLike(s) => s.fmt(f),
+            Statement::DropTable(s) => s.fmt(f),
+            Statement::DropDatabase(s) => s.fmt(f),
+            Statement::CreateDatabase(s) => s.fmt(f),
+            Statement::Alter(s) => s.fmt(f),
+            Statement::ShowDatabases(s) => s.fmt(f),
+            Statement::ShowTables(s) => s.fmt(f),
+            Statement::ShowColumns(s) => s.fmt(f),
+            Statement::ShowIndex(s) => s.fmt(f),
+            Statement::ShowCreateTable(s) => s.fmt(f),
+            Statement::DescribeTable(s) => s.fmt(f),
+            Statement::Explain(s) => s.inner.fmt(f),
```

Comment:
```suggestion
            Statement::Explain(s) => s.fmt(f),
```

Explain implement Display already. Although it's `write!(f, "{}", self.inner)` inside so the same of this line physically, we'd better call the implementation on our Explain so that further changes underneath don't break this line.

Reply:
Resolved.